cashier-iab/src/main/java/com/getkeepsafe/cashier/iab/InAppBillingV3Vendor.java

```diff
@@ -189,6 +189,8 @@ public boolean available() {
     return available && api.available() && canPurchaseAnything();
   }
 
+  int getRequestCode() {  return requestCode; }
```

Comment:
Please make this in a new line and follow code style from the rest of the app

Reply:
yes, surprise even for me that I've written it like that ))